apps/bfd-server/bfd-server-war/src/test/resources/openapi/posts.yaml

```diff
@@ -122,18 +121,16 @@ paths:
         '200':
           description: Success
           content:
-            application/fhir+json:
-              schema: {$ref: '#/components/schemas/FHIR-JSON-RESOURCE'}
-            application/fhir+xml:
-              schema: {$ref: '#/components/schemas/FHIR-XML-RESOURCE'}
+            application/x-www-form-urlencoded:
+              schema:
```

Comment:
These code the content of the response rather than the request–I don't think these need to be adjusted here or elsewhere under the content sections.

Reply:
fixed